src/main/java/seedu/foodrem/commons/enums/CommandType.java

```diff
@@ -190,6 +190,19 @@ public String getUsage() {
                     + getCommandWord() + " 1";
         }
     },
+    FILTER_TAG_COMMAND("filtertag") {
+        @Override
+        public String getUsage() {
+            return getCommandWord() + ": Filters all items in FoodRem by the specified tag"
+                    + "\n\n"
+                    + "Format:\n"
+                    + getCommandWord() + " "
+                    + PREFIX_NAME + "NAME\n\n"
+                    + "Example:\n"
+                    + getCommandWord() + " "
+                    + PREFIX_NAME + "Potatoes ";
+        }
+    },
```

Comment:
Can we insert this in the appropriate location following alphabetical order, like the rest?

Reply:
Oops, this was merged already.